.buildkite/pipeline.yml

```
@@ -136,6 +136,50 @@ steps:
 
   # ----------
 
+  # Run CVE checks for the main branch
+  - group: cve image scans
+    steps:
+
+      # scan the operator image for CVEs on merge to main or nightly builds
+      - label: ":buildkite: cve-checks"
+        if: build.branch == "main"
+        depends_on:
+          - "operator-image-build"
+        command: |
+          export CONTAINER=$$(buildkite-agent meta-data get operator-image)
+          cat <<- YAML | buildkite-agent pipeline upload
+          steps:
+            - label: "Trigger cve-slo-status pipeline for $$CONTAINER"
+              trigger: cve-slo-status
+              build:
+                env:
+                  CONTAINER: "$$CONTAINER"
+              soft_fail: true
+          YAML
+        agents:
+          image: docker.elastic.co/ci-agent-images/cloud-k8s-operator/buildkite-agent:e808dbe5
+          memory: "2G"
```

Could we ask for less? With a real test, we could look at the memory usage.
```suggestion
          memory: "128Mi"
```

---

👍 , looks like the previous PR's build (https://buildkite.com/elastic/cve-slo-status/builds/35124) metrics might be no longer reliable, so I'll run the check on the PR to get current metrics.

---

Thanks, I see it used [`2.4% x 2Gi = 49Mi`](https://buildkite-stats.elastic.dev/app/metrics/detail/pod/5e832ebf-81a2-4253-890c-67b9604ce35b?_a=(autoReload:!f,refreshInterval:5000,time:(from:%272024-11-05T19:05:30.180Z%27,interval:%3E%3D1m,to:%272024-11-05T19:05:53.719Z%27))).